core/embed/trezorhal/stm32u5/secret.c

```diff
@@ -9,7 +9,7 @@
 static secbool bootloader_locked_set = secfalse;
 static secbool bootloader_locked = secfalse;
 
-static secbool verify_header(void) {
+secbool secret_verify_header(void) {
   uint8_t header[SECRET_HEADER_LEN] = {0};
```

Comment:
kinda feeling that you should memzero this at the end?
or maybe just copy `sizeof(SECRET_HEADER_MAGIC)` bytes because the secret doesn't seem to be used here either

Reply:
https://github.com/trezor/trezor-firmware/commit/5bbddd37420c278a03dd8f333913b36b26eeb49d

(the header does not contain any secret info)